adminer/lang/cs.inc.php

```diff
@@ -354,6 +354,7 @@
 	'%s must <a%s>return an array</a>.' => '%s musí <a%s>vracet pole</a>.',
 	'<a%s>Configure</a> %s in %s.' => '<a%s>Nakonfigurujte</a> %s v %s.',
 	'screenshot' => 'obrázek',
+	'Index Method' => 'Typ Indexu',
```

Comment:
Small `m` in English, small `i` in Czech. I wanted you to update [xx.inc.php](https://github.com/vrana/adminer/blob/master/adminer/lang/xx.inc.php), you don't need to learn Czech 😉.